src/drivers/uavcan/uavcan_drivers/kinetis/driver/src/uc_kinetis_flexcan.cpp

```diff
@@ -693,7 +693,7 @@ void CanIface::handleRxInterrupt(uavcan::uint32_t rx_iflags, uavcan::uint64_t ut
 		frame.data[7] = rf.data.b7;
 
 		volatile uavcan::uint32_t idhit = can_->RXFIR;
-		UNUSED(idhit);
+		//UNUSED(idhit);
```

Comment:
Why not just remove this?

Reply:
No reason, just wasn't treating it like "our" code.
It was also kind of a peculiar error.

Removed.